app/components/feedback/Feedback.tsx

```diff
@@ -0,0 +1,57 @@
+import { useState } from "react"
+import { FormProps } from "../../constants/types/components_props/types"
+import { SEND_FEEDBACK_EMAIL } from "../../services/api/variables"
+import styles from "../../styles/components/feedback.module.scss"
+import { handleInput, handleValidation } from "../../services/utils/validationHandlers"
+import sendFeedbackMessage from "../../services/form_services/feedback_form/feedback-form.service"
+
+const Feedback = ({ text, endpoint = SEND_FEEDBACK_EMAIL }: FormProps) => {
+  const [messageResponse, setMessageResponse] = useState<string>("")
```

Comment:
When we use useState and assign a default value like this const = [ isloading, setLoading ] = useState(false) we are recommended by best practices in typescript that the type is not necessary, because it is inferred the type.